src/ResourceManager/ContainerInstance/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Current Release
 
+## Version 0.2.4
```

Comment:
@yolo3301 you don't need this version header -- you can just put this content under the `## Current Release` header